src/contracts/treasury/Collector.sol

```diff
@@ -83,12 +94,12 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
   /*** Contract Logic Starts Here */
 
   /// @inheritdoc ICollector
-  function initialize(address fundsAdmin, uint256 nextStreamId) external initializer {
+  function initialize(address aclManager, uint256 nextStreamId) external initializer {
```

Comment:
probably makes sense to make this one virtual, so you can overwrite on a proposal